.github/workflows/unit_test.yml

```diff
@@ -0,0 +1,51 @@
+name: PR Build - Unit Tests
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  unit_test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: gradle/wrapper-validation-action@v1
+      # The first JDK set up gets to be "primary".
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1.4.0
+        with:
+          java-version: 8
+      - name: save JAVA_HOME as JDK8 for later usage
+        run: echo "::set-env name=ORG_GRADLE_PROJECT_jdk8::$JAVA_HOME"
+      # The second JDK is not first in the path but does reset JAVA_HOME
+      - name: Set up JDK 1.7
+        uses: actions/setup-java@v1.4.0
+        with:
+          java-version: 7
+      - name: save JAVA_HOME as JDK7 for later usage
+        run: echo "::set-env name=ORG_GRADLE_PROJECT_jdk7::$JAVA_HOME"
+      # Java is installed; document settings
+      - name: log project settings
+        run: |
+          echo "--- Java versions ---"
```

Comment:
Confirming the reason for setting jdk7 and jdk8 gradle properties is because `java.gradle` script looks for them?

Reply:
Yes. In fact, it requires those two properties.